AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -92,8 +92,8 @@ - (instancetype)init {
 
     // Proceed update whenever an application is restarted in users perspective.
     [MS_NOTIFICATION_CENTER addObserver:self
-                               selector:@selector(applicationWillEnterForeground)
-                                   name:UIApplicationWillEnterForegroundNotification
+                               selector:@selector(applicationDidBecomeActive)
+                                   name:UIApplicationDidBecomeActiveNotification
```

Comment:
I would add a comment right in the code that we can't reliable send network requests from `applicationWillEnterForeground`

Reply:
👌